inc/audiences/rest_api/namespace.php

```diff
@@ -268,3 +268,13 @@ function check_edit_permission() : bool {
 	$type = get_post_type_object( Audiences\POST_TYPE );
 	return current_user_can( $type->cap->edit_posts );
 }
+
+/**
+ * Check user can view audience posts.
+ *
+ * @return bool
+ */
+function check_read_permission() : bool {
+	$type = get_post_type_object( Audiences\POST_TYPE );
+	return current_user_can( $type->cap->read );
```

Comment:
Let’s use the primitive `edit_posts` capability here instead. I don’t think we want the general public to be able to see what kind of traffic the site is getting. Contributors and up will be able to see them.

Reply:
If I have access to work with pages on a site, but not blog posts (i.e. `edit_pages` but not `edit_posts`), presumably I should have access to audiences as well for the same reasoning? Seems to me like using the read cap for audiences is the better path, and then mapping the meta cap under the hood if we want to default it.

Reply:
I'd rather use a custom capability than default to read in that case, then as you say use `map_meta_cap`. My mistake there though, I thought this was the primitive. Will need to update this in the same for viewing XB analytics data in that case too. I'll make a PR for that on the experiments repo.

Reply:
To be clear, when I say read, I mean `$type->cap->read`, not the primitive `'read'`. That can be set to `read_audiences` in the post type registration, then `map_meta_cap` can map that to whatever under the hood

Reply:
Audience posts themselves are public, this is specifically for the analytics data related to audiences

Reply:
🤔

Reply:
I was wrong actually, this permission check is fine and doesn't need the same permissions as the xb analytics. I think I got the wrong end of the stick there so we should probably use the same check that we have here in the experiments plugin - if you can read audiences which are not actually public (was getting confused because we output those programmatically on the front end) then you should be able to read the analytics data for XBs.

I'll make an issue to correct it on the experiments repo to use the audience read permission..